jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -3705,7 +3705,7 @@ private static boolean counterPredicate(int endValue, int counter) {
 	}
 	
 	/**
-	 * Produce a loop handle that iterates over a range of values produced by an Iterator<T>
+	 * Produce a loop handle that iterates over a range of values produced by an Iterator object
```

Comment:
We want to keep the `<T>` generic.  Changing it to `Iterator object` changes the meaning.  Maybe wrap it in `{@code Iterator<T> }`

Reply:
Hi Dan, yes I have done all those changes.. some how it is not seen or refresh those latest changes here .. wired .. I am checking  on it. you will see latest one .. these are bit old changes then I made everything with {@code}